automation/alpha-archived-metadata-file-upload-check.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -euo pipefail
+
+VAULT_TOKEN=$(cat /etc/vault-token-dsde)
+
+# Get a workflow which completed at least 40 minutes ago and whose metadata is archived
+# (archive-delay in alpha is set to 30 minutes. Hence adding 10 more minutes should give Cromwell enough time to
+# archive such a workflow)
+ARCHIVED_WORKFLOW_RESULT=$(
+  docker run --rm \
+     -e VAULT_TOKEN=$VAULT_TOKEN \
+     broadinstitute/dsde-toolbox mysql-connect.sh -p firecloud -a cromwell1 -e alpha \
+     "SELECT WORKFLOW_EXECUTION_UUID, ROOT_WORKFLOW_EXECUTION_UUID FROM cromwell.WORKFLOW_METADATA_SUMMARY_ENTRY WHERE METADATA_ARCHIVE_STATUS = 'Archived' AND END_TIMESTAMP <= DATE_SUB(NOW(), INTERVAL 40 minute) ORDER BY END_TIMESTAMP DESC LIMIT 1;"\
```

Comment:
Stylistic only, the `-` operator is overloaded to work as one expects
```suggestion
     "SELECT WORKFLOW_EXECUTION_UUID, ROOT_WORKFLOW_EXECUTION_UUID FROM cromwell.WORKFLOW_METADATA_SUMMARY_ENTRY WHERE METADATA_ARCHIVE_STATUS = 'Archived' AND END_TIMESTAMP <= ( NOW() - INTERVAL 40 minute ) ORDER BY END_TIMESTAMP DESC LIMIT 1;"\
```